WcaOnRails/app/views/regulations/scrambles/index.html.erb

```diff
@@ -18,6 +18,16 @@
     <li>Delegates should download TNoodle to run it on a computer. They should not use TNoodle running on a public server (for security reasons).</li>
     <li>Delegates must save all scramble sequences generated for an official competition, and send them with the results (see <%= link_to "Regulation 1c3a", "../#1c3a" %>).</li>
   </ul>
+  <h3 id="scramble-secrecy">Scramble Secrecy</h3>
+    <ul>
+      <li><b>Always</b> encrypt your scrambles in TNoodle with an unguessable password. The password must have nothing to do with the competition or personal data (name, birthdate) of any of the Delegates.</li>
```

Comment:
nit: Is this the first time we've addressed delegates using "you" in this doc? If so, maybe worth rephrasing?